pkg/authenticator/authenticator.go

```diff
@@ -236,27 +248,25 @@ func (auth *Authenticator) ParseAuthenticationResponse(response []byte) error {
 		content = response
 	}
 
-	// InfoLogger.Printf("Writing token %v to shared volume ...", content)
-	err = ioutil.WriteFile(auth.Config.TokenFilePath, content, 0644)
+	err = auth.AccessTokenHandler.Write(content)
```

Comment:
Can we rename the field `AccessTokenHandler` on `auth` to `AccessToken` ? To be consistent.

Reply:
nice catch. i thought i already did this...